src/util/d.ts

```diff
@@ -16,28 +16,33 @@ export type DOptions<S extends WidgetState, O extends WidgetOptions<S>> = VNodeP
 
 type Children = (DNode | VNode | null)[];
 
-function d(tagName: string, options?: VNodeProperties, children?: (DNode | VNode | null)[]): HNode;
+function d(tagName: string, optionsOrChildren?: VNodeProperties | Children, children?: Children): HNode;
```

Comment:
from a usability perspective, it might be nicer to have a better set of overloads here, it will give cleared intellisense:

```ts
function d(tagName: string, options: VNodeProperties, children?: Children): HNode;
function d(tagName: string, children: Children): HNode;
function d(tagName: string): HNode;
```

You may also need to export `Children` now so it can be "named" when creating the declaration for `d`.  I would check with a `grunt dist`.